electron/config.ts

```diff
@@ -173,20 +173,22 @@ abstract class GlobalConfig {
     } else if (!isWindows) {
       // Linux
       const crossoverWineBin = '/opt/cxoffice/bin/wine'
-      const crossoverVersion = (
-        await execAsync(crossoverWineBin + ' --version')
-      ).stdout
-        .toString()
-        .split('\n')[2]
-        .split(':')[1]
-        .trim()
       if (existsSync(crossoverWineBin)) {
-        crossover.add({
-          bin: `'${crossoverWineBin}'`,
-          name: `CrossOver - ${crossoverVersion}`,
-          type: 'crossover',
-          ...this.getWineExecs(crossoverWineBin)
-        })
+        const crossoverVersion = (
```

Comment:
Arg, yup, this one's my fault. Thanks for catching that

Reply:
also, just a note: when using `await` is good to put it inside a try/catch block.

Reply:
This is now superseded by https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1046